src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -54,6 +56,23 @@ namespace
 {
 
 template<typename T>
+std::string maskedCompoundNumericPlugRepr( const T *plug, unsigned flagsMask )
+{
+	std::string extraArgs = "";
+
+	const IECore::GeometricData::Interpretation interpretation = plug->interpretation();
+	if( interpretation != IECore::GeometricData::None && interpretation != IECore::GeometricData::Color )
```

Comment:
Dan suggested to set the interpretation to None for Color3fPlugs explicitly. So maybe detect Color here [0] and set it to None?

[0] https://github.com/mattigruener/gaffer/blob/26fe8bc7510c18e4e6f8f72658e575cdfc5de655/src/GafferOSL/OSLShader.cpp#L450